kernel/futex.c

```diff
@@ -1533,6 +1544,10 @@ futex_wake(u32 __user *uaddr, unsigned int flags, int nr_wake, u32 bitset)
 	if (!hb_waiters_pending(hb))
 		goto out_put_key;
 
+	sched_log_trace(SCHED_LOG_WAKER_FUTEX, task_cpu(current), current,
+			(unsigned long)hb >> 32,
```

Comment:
We should try to use the user address (uaddr) instead of the kernel-allocated hb object.